caterva2/api.py

```diff
@@ -628,12 +629,26 @@ def deluser(user, urlbase=None, auth_cookie=None):
     auth_cookie : str
         A HTTP cookie for authorizing access.
         This must be specified unless it was already specified inside
-        a :py_obj:`caterva2.c2context`.
+        a :py_obj:`caterva2.c2context`. The auth_cookie used must be
+        from a superuser.
 
     Returns
     -------
     str
         An explanatory message.
+
+    Examples
+    --------
+    >>> import caterva2 as cat2
+    >>> import numpy as np
+    >>> # To delete a user you need to be a superuser
+    >>> # This example is intended to work when the subscriber is running locally
+    >>> super_auth_cookie = cat2.get_auth_cookie(cat2.sub_urlbase_default, user_auth=dict(username='superuser@example.com', password='foo'))
```

Comment:
Line is too long

Reply:
Fixed in 0fe79ea0f27976e7d428308106ade05bc649e08b